tests/connectors/test_druid.py

```diff
@@ -60,6 +63,21 @@ def test_fetch_data(self):
         )
         self.assertEqual(_out.shape, (2, 2))
 
+    def test_build_param(self):
+        expected = {
+            "datasource": "foo",
+            "dimensions": map(lambda d: DimensionSpec(dimension=d, output_name=d), ["bar"]),
+            "filter": [Filter(type="selector", dimension="ciStatus", value="false")],
+            "granularity": "all",
+            "aggregations": "",
+            "intervals": "",
+        }
+
+        filter_pairs = make_filter_pairs(["ciStatus"], ["false"])
+        actual = build_params("", "foo", ["bar"], filter_pairs, "all", float(24))
+        actual["intervals"] = ""
+        DeepDiff(expected, actual).get("values_changed", {})
```

Comment:
Are we doing an assert here? If `values_changed` is non zero, will this test fail?